website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
@lasley I vote for keeping company + author> Company is the legal thing. Now author is the technical guy to contact and it's a reward for the work. Don't forget that all this code is given away for free, but I think it's important for our ecosystem to educate companies that they lower their risk and support the ecosystem if they hire the people who write the code instead of the marketing middlemen. Authors can change companies and it's important their work can still be tracked, file by file. IMHO we should keep the contributors of the README for those who contributed a very significant proportion of the module otherwise nothing is trackable and the meritocracy is broken.